files/build_templates/docker_image_ctl.j2

```diff
@@ -646,6 +705,9 @@ stop() {
     /usr/local/bin/container stop -t 60 $DOCKERNAME
     {%- else %}
     /usr/local/bin/container stop $DOCKERNAME
+    {%- if docker_container_name == "snmp" %}
+    docker rm $DOCKERNAME
```

Comment:
>docker rm [](http://example.com/codeflow?start=4&length=9)

Why remove docker container?

Reply:
If there is hot patches inside docker container, original behavior is that the patches will survive config reload or reboot. But this PR lose the capabilities.

Reply:
Are there real use-cases where a docker is being loaded with patches inside it?
There might also be some local files (such as temp config and such in every container), is this a real use-case?

Reply:
Yes, it is possible in production environment.

Reply:
Still applicable.